src/OnePlusBot/Base/Core.cs

```diff
@@ -40,7 +40,7 @@ public async Task MainAsync()
             
             if (!File.Exists("messageid.txt"))
             {
-                Console.WriteLine("You need a message.txt, role reactions won't work otherwise.");
+                Console.WriteLine("You need a messageid.txt file, role reactions won't work otherwise.");
```

Comment:
If it doesn't exist, create it, as mentioned in 71445b5d39c3e55a384ffe79c8812f95e140ac10 comments.